src/tests/ftest/osa/offline_drain.py

```diff
@@ -109,28 +109,29 @@ def run_offline_drain_test(self, num_pool, data=False, oclass=None, pool_fillup=
                     continue
                 output = self.pool.drain(rank, t_string)
                 self.print_and_assert_on_rebuild_failure(output)
-                free_space_after_drain = self.pool.get_total_free_space(refresh=True)
+                total_space_after_drain = self.pool.get_total_space(refresh=True)
 
                 pver_drain = self.pool.get_version(True)
                 self.log.info("Pool Version after drain %d", pver_drain)
                 # Check pool version incremented after pool drain
-                self.assertTrue(pver_drain > (pver_begin + 1), "Pool Version Error:  After drain")
+                self.assertGreater(pver_drain, (pver_begin + 1),
+                                   "Pool Version Error:  After drain")
                 if self.test_during_aggregation is False:
-                    self.assertTrue(initial_free_space > free_space_after_drain,
-                                    "Expected free space after drain is less than initial")
+                    self.assertGreater(initial_total_space, total_space_after_drain,
+                                       "Expected total space after drain is more than initial")
                 if num_pool > 1:
                     output = self.pool.reintegrate(rank, t_string)
                     self.print_and_assert_on_rebuild_failure(output)
-                    free_space_after_reintegration = self.pool.get_total_free_space(refresh=True)
-                    self.assertTrue(free_space_after_reintegration > free_space_after_drain,
-                                    "Expected free space after reintegration is less than drain")
+                    total_space_after_reintegration = self.pool.get_total_space(refresh=True)
+                    self.assertGreater(total_space_after_reintegration, total_space_after_drain,
+                            "Expected total space after reintegration is less than drain")
```

Comment:
I'd recommend putting all args on a new line to avoid indentation games
```suggestion
                    self.assertGreater(
                        total_space_after_reintegration, total_space_after_drain,
                        "Expected total space after reintegration is less than drain")
```